src/run_reps.jl

```diff
@@ -0,0 +1,134 @@
+using Dates
+using Random
+
+"""
+    run_rme(rme_path::String, n_threads::Int64, reps::Int64, result_path::String; start_year::Int64=2022, end_year::Int64=2099, batch_size::Int64=10, start_batch::Int64=1, RCP_scen::String="SSP 2.45", gcm::String="CNRM_ESM2_1", rnd_seed::Int64=1234)::Nothing
+
+Run counterfactual scenarios with ReefModEngine.jl and save result set to desired dir.
+
+# Arguments
+- `rme_path` : Path to REM folder.
+- `n_threads` : Number of threads to be used with RME.
+- `reps` : Total number of repetitions to be run.
+- `result_path` : Path to folder where resultset should be placed.
+- `start_year` : RME run start year.
+- `end_year` : RME run end year.
+- `batch_size` : Number of repetitions to be run in each batch.
+- `RCP_scen` : RCP scenario to be used for RME runs.
+- `gcm` : GCM to be used for RME runs.
+- `rnd_seed` : Random seed.
```

Comment:
Prefer to accept an RNG instead:

```suggestion
- `rng` : Random number generator (default: Globally set RNG).
```

`rng::AbstractRNG=Random.GLOBAL_RNG`